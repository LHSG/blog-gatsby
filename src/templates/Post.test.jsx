import React from 'react';
import { shallow } from 'enzyme';

import PostTemplate from './Post';
import { sitePropType, markdownRemarkPropType } from '../../test/fixtures';

test('rendering', () => {
  const props = {
    data: {
      site: sitePropType,
      markdownRemark: markdownRemarkPropType,
    },
  };
  const wrapper = shallow(<PostTemplate {...props} />);
  expect(wrapper).toMatchSnapshot();
});
