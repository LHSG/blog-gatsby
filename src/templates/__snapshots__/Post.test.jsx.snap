// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering 1`] = `
<div>
  <Head
    excerpt="excerpt"
    image=""
    path="path"
    site={
      Object {
        "siteMetadata": Object {
          "author": "author",
          "description": "description",
          "facebookAppId": "facebookAppId",
          "siteUrl": "siteUrl",
          "social": Object {
            "githubUrl": "githubUrl",
            "keybaseUrl": "keybaseUrl",
            "linkedInUrl": "linkedInUrl",
            "twitterUrl": "twitterUrl",
          },
          "title": "title",
          "twitterUser": "twitterUser",
        },
      }
    }
    title="title"
  />
  <Post
    coverImageSizes={
      Object {
        "sizes": "something",
      }
    }
    date="date"
    excerpt="excerpt"
    html="<div>some content</div>"
    title="title"
  />
</div>
`;
